<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceWidget</class>
 <widget class="QWidget" name="ReferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="createRefLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create a new Reference:</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="newRefSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The publication to cite:</string>
     </property>
     <property name="buddy">
      <cstring>publicationEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ContactEdit" name="publicationEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="publicationListButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Citekey:</string>
       </property>
       <property name="buddy">
        <cstring>citeKeyEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="StringEdit" name="citeKeyEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pages:</string>
       </property>
       <property name="buddy">
        <cstring>pagesEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="StringEdit" name="pagesEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Chapter</string>
       </property>
       <property name="buddy">
        <cstring>chapterEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="StringEdit" name="chapterEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create Reference</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>remove Reference</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/contactedit.h</header>
  </customwidget>
  <customwidget>
   <class>StringEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/stringedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>publicationListButton</sender>
   <signal>clicked()</signal>
   <receiver>ReferenceWidget</receiver>
   <slot>showPublicationList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>ReferenceWidget</receiver>
   <slot>createReference()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>ReferenceWidget</receiver>
   <slot>removeReference()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showPublicationList()</slot>
  <slot>createReference()</slot>
  <slot>removeReference()</slot>
 </slots>
</ui>
